eslint.config.mjs

```diff
@@ -0,0 +1,78 @@
+import nextPlugin from '@next/eslint-plugin-next'
+import js from '@eslint/js'
+import tseslint from 'typescript-eslint'
```

Comment:
`next/typescript` configuration is based on plugin:@typescript-eslint/recommand

Since `plugin:@typescript-eslint/recommended` is a legacy config, when using flat config you can replace it with:

export default tseslint.config(...tseslint.configs.recommended);

Refs: https://typescript-eslint.io/users/configs/#recommended